### PR TITLE
NY_FSS missing

### PR DESCRIPTION
**Pull request by Connorseds2002**
Edit to fix NY_FSS missing

### CODE REVIEW

**Comment by Connorseds2002** [comment]
Fixed the VATSpy.data file to add NY_FSS

**Comment by NelisV** [comment]
image
Had a look at these changes:
**A**, Overlap with KZMA, better to adjust that area too. 
**B**, Gap between KZNY and TJZS created. 

I'd have to dive in, but i recall this was intentionally changed to KZWY a while ago, is this change aligned with the guys in NY?

File: VATSpy.dat [hunk 1]
```diff
@@ -17675,7 +17675,6 @@ ZYQQ|Qiqihar|47.38306|123.91639||ZYSH|0
 ZYTL|Dalian|38.965556|121.538333||ZYSH|0
 ZYTX|Shenyang|41.638333|123.485||ZYSH|0
 ZYYJ|Yanji|42.8828|129.451258||ZYSH|0
-
```
Comment (NelisV) [inline VATSpy.dat:17678]: typo or intentional?

**Comment by Connorseds2002** [comment]
Thanks for the reply will take a look in to it for you and see what it up with that 

 

Connor Sedman

S2 Controller 

IT Developer - VATSIM Australia Pacific

E: c.sedman@vatpac.org <mailto:c.sedman@vatpac.org>

**Comment by Connorseds2002** [comment]
Thanks will update the file to have the correct A and B added as well 

 

Thanks for letting me know that this was missing from the file 

 

Connor Sedman

S2 Controller 

IT Developer - VATSIM Australia Pacific

E: c.sedman@vatpac.org <mailto:c.sedman@vatpac.org>

File: VATSpy.dat (hunk 1 above)
Comment (Connorseds2002) [reply VATSpy.dat:17678]: Not sure what happened I think it was a typo and will be updated @NelisV

**Comment by Connorseds2002** [comment]
Hi there thanks for letting me know about this I have fixed the errors that I could see be let me know if there is anything that does not look right 

thanks
Connor
[c.sedman@vatpac.org](mailto:c.sedman@vatpac.org)

**Comment by Connorseds2002** [comment]
image
fixed as I can see from this image 

Thanks
Connor
[c.sedman@vatpac.org](mailto:c.sedman@vatpac.org)

**Comment by Connorseds2002** [comment]
> image
> Had a look at these changes:
> **A**, Overlap with KZMA, better to adjust that area too.
> **B**, Gap between KZNY and TJZS created.
> 
> I'd have to dive in, but i recall this was intentionally changed to KZWY a while ago, is this change aligned with the guys in NY?

as I could see from there website there where stall using the same callsign as the center position bit will _FSS at the end to demote as oceanic

**Comment by Connorseds2002** [comment]
this to do the issue #152

**Review by NelisV — APPROVED** [review]
Looks good now!

**Review by prryplatypus — APPROVED** [review]

**Pull request merged**